_orders/querying.md

```diff
@@ -0,0 +1,498 @@
+  "limit" : 100,
+  "nextMarker" : "6705424597053369344",
+  "results" : [ {
+    "Completed" : "2024-02-06T20:49:44.000Z",
+    "Created" : "2024-02-06T20:49:42.000Z",
+    "Deleted" : false,
+    "DossierGuid" : "97339714918566161680",
+    "Fulfilled" : "2024-02-06T20:49:42.000Z",
+    "OrderGuid" : "2122759493555823372",
+    "OrderSummary" : "Cyril Figgins 2",
+    "OwnerCompanyName" : "Bloggins Bank",
+    "OwnerUserContactMechanism" : "joe@blogginsbank.com",
+    "OwnerUserDisplayName" : "Bloggins, Joe",
+    "ProductDisplayName" : "ADV-AB-BeforeReconfirm",
+    "ProductType" : "ADV",
+    "ReferenceNumber" : "7058774124",
+    "ReportAvailable" : true,
+    "Status" : "complete"
+  }, ... ]
+}
+```
+
+### Step 4: Find orders by Reference Number and limit the number of results to 5
+
+```bash
+curl -X POST https://api.pointservices.com/riskinsight-services-ws/resources/v1/orders/query 
+-H "Authorization: Bearer your_access_token_here" 
+-H "Content-Type: application/json" 
+-d '{"filters":{"ReferenceNumber":{"eq":"7058774124"}},"limit":5}'
+```
+
+Example Response
+
+```json
+{
+  "count" : 1,
+  "limit" : 5,
+  "nextMarker" : "6705424597053369344",
+  "results" : [ {
+    "Completed" : "2024-02-06T20:49:44.000Z",
+    "Created" : "2024-02-06T20:49:42.000Z",
+    "Deleted" : false,
+    "DossierGuid" : "97339714918566161680",
+    "Fulfilled" : "2024-02-06T20:49:42.000Z",
+    "OrderGuid" : "2122759493555823372",
+    "OrderSummary" : "Cyril Figgins 2",
+    "OwnerCompanyName" : "Bloggins Bank",
+    "OwnerUserContactMechanism" : "joe@blogginsbank.com",
+    "OwnerUserDisplayName" : "Bloggins, Joe",
+    "ProductDisplayName" : "ADV-AB-BeforeReconfirm",
+    "ProductType" : "ADV",
+    "ReferenceNumber" : "7058774124",
+    "ReportAvailable" : true,
+    "Status" : "complete"
+  }]
+}
+```
+
+### Step 5: Find orders greater than a specific date after the maker and limit the results to 5
+
+```bash
+curl -X POST https://api.pointservices.com/riskinsight-services-ws/resources/v1/orders/query 
+-H "Authorization: Bearer your_access_token_here" 
+-H "Content-Type: application/json" 
+-d '{"filters":{"Completed":{"gt":"2024-02-06T20:49:43.000Z"}},"limit":5, "marker":"6705424597053369343"}
+```
+
+Example Response
+
+```json
+{
+  "count" : 1,
+  "limit" : 5,
+  "nextMarker" : "6705424597053369344",
+  "results" : [ {
+    "Completed" : "2024-02-06T20:49:44.000Z",
+    "Created" : "2024-02-06T20:49:42.000Z",
+    "Deleted" : false,
+    "DossierGuid" : "97339714918566161680",
+    "Fulfilled" : "2024-02-06T20:49:42.000Z",
+    "OrderGuid" : "2122759493555823372",
+    "OrderSummary" : "Cyril Figgins 2",
+    "OwnerCompanyName" : "Bloggins Bank",
+    "OwnerUserContactMechanism" : "joe@blogginsbank.com",
+    "OwnerUserDisplayName" : "Bloggins, Joe",
+    "ProductDisplayName" : "ADV-AB-BeforeReconfirm",
+    "ProductType" : "ADV",
+    "ReferenceNumber" : "7058774124",
+    "ReportAvailable" : true,
+    "Status" : "complete"
+  }]
+}
+```
+
```

Comment:
Right here, before the DSL, I suggest we have 2 new sections.  One is a documentation to describe what the request json will look like.  Specifically, you haven't yet mentioned that limit is 100 when certain conditions are met.  The 2nd is the documentation to describe response.  Specifically count is the number of results returned in this response, and not total number of results.  

I'm actually not sure if we should be returning "Deleted" or not.  Maybe the name should be changed to 'Archived' instead?